src/components/Dashboard/FilingHistoryList/CompletedAlteration.vue

```diff
@@ -8,9 +8,9 @@
       on <DateTooltip :date="filing.effectiveDate" />.
     </p>
 
-    <p v-if="filing.courtOrderNumber">Court Order Number: {{filing.courtOrderNumber}}</p>
+    <p v-if="filing.courtOrderNumber" class="mb-0">Court Order Number: {{filing.courtOrderNumber}}</p>
 
-    <p v-if="filing.isArrangement">Pursuant to a Plan of Arrangement</p>
+    <p v-if="filing.isArrangement" class="mt-0">Pursuant to a Plan of Arrangement</p>
```

Comment:
All paragraphs in all detail components have top and bottom margin as 1rem. This is the exception, since we want both lines to have no extra whitespace between them.